docs/development/observability/logging.mdx

```diff
@@ -10,12 +10,35 @@ The `SERVER_VERBOSITY` variable accepts the following values: `DEBUG`, `INFO`, `
 
 ## Server logging format
 
+You can customize the format by setting the `SERVER_LOGGING_FORMAT` environment variable. By default it is set as
+`SERVER_LOGGING_FORMAT=plaintext`. If it is set to unknown format it will use default value instead.
+
+### Plaintext format
+
+The plaintext logging format will be used by default. Or it can be set explicitly by setting the
+`SERVER_LOGGING_FORMAT=plaintext` environment variable.
+
 By default, the server will log messages in the following format:
 ```
-[%(levelname)s]:[%(name)s]:%(message)s
+%(levelname)s:%(name)s:%(message)s
 ```
 
-You can customize the format by setting the `SERVER_LOGGING_FORMAT` environment variable.
+It can be customized by setting the `SERVER_LOGGING_FORMAT` environment variable.
+Example:
+```
+SERVER_LOGGING_FORMAT=$'%(asctime)s\t[%(name)25.25s]\t%(levelname)5s: %(message)s'
+```
+
+**Important!**
```

Comment:
you can use a <Warning> block like here: https://github.com/mage-ai/mage-ai/blob/154cbc2460e1560f251d7d993dd71a66e9882bc2/docs/guides/blocks/r-blocks.mdx#L9

Reply:
Done